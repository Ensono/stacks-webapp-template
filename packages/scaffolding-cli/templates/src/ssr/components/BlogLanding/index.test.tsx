import {render} from "@testing-library/react"
import React from "react"
import {BlogLanding} from "."

test("renders BlogLanding snapshot", () => {
    const {asFragment} = render(<BlogLanding posts="" />)
    expect(asFragment()).toMatchSnapshot()
})
