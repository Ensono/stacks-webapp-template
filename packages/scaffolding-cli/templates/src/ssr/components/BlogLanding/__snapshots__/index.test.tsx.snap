// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders BlogLanding snapshot 1`] = `
<DocumentFragment>
  <h2
    class="MuiTypography-root MuiTypography-h5"
  >
    Yumido Blog
  </h2>
  <br />
</DocumentFragment>
`;
